tests/js/server/dump/dump-modified.js

```diff
@@ -277,6 +277,74 @@ function dumpTestSuite () {
       }
     },
 
+////////////////////////////////////////////////////////////////////////////////
+/// @brief test padded keygen
+////////////////////////////////////////////////////////////////////////////////
+
+    testKeygenPadded : function () {
```

Comment:
The same method I've just reviewed in "tests/js/server/dump/dump-mixed-cluster.js". 
Please re-use that method here. This is hard to maintain.